app/Tasks/Types/TheoreticalExam.php

```diff
@@ -41,6 +43,14 @@ public function create(Task $model)
 
     public function complete(Task $model)
     {
+        if (Vatsim::divisionIs('EUD')) {
+            $response = Http::post('https://core-dev.vateud.net/api/assign', [
+                'user_cid' => $model->subject_user_id,
+                'exam_id' => 1,
+                'instructor_cid' => $model->assignee_user_id,
+            ]);
+        }
+
```

Comment:
this would go away in lieu of using the contract directly, assuming the controller that instantiates the task resolves the dependency, if I understand the Laravel service container correctly.

Reply:
Yeah, this part I didn't yet adjust to the new service